src/app/lib/config/services/news.js

```diff
@@ -23,6 +23,7 @@ const news = {
   noBylinesPolicy: 'https://www.bbc.com/news/help-41670342#authorexpertise',
   publishingPrinciples: 'https://www.bbc.com/news/help-41670342',
   script: latin,
+  link: 'https://bbc.com/news',
```

Comment:
needs www.

Reply:
Checked and @pjlee11 supports having the option of both .co.uk and .com

Working on adding that